corehq/motech/dhis2/entities_helpers.py

```diff
@@ -67,6 +87,23 @@ def send_dhis2_entities(requests, repeater, case_trigger_infos):
     return RepeaterResponse(200, "OK")
 
 
+def _get_tracked_entity_relationship_specs(tracked_entity):
+    """Returns a list of RelationshipSpec objects that represents the relationships that already exist on a tracked
+    entity instance"""
+    if not tracked_entity:
+        return []
+
+    tracked_entity_relationships = []
+    for relationship in tracked_entity.get("relationships", []):
+        spec = RelationshipSpec(
```

Comment:
These object creations can be done using `__init__` with less verbosity to avoid the attribute repitition.

It could become
```
tracked_entity_relationships.append(
    RelationshipSpec(
           relationship["relationshipType"],
           relationship["to"]["trackedEntityInstance"]["trackedEntityInstance"],
           relationship["from"]["trackedEntityInstance"]["trackedEntityInstance"]
    )
)
```

Reply:
I always prefer to see which variable/value is assigned to which attribute. It's too easy to accidentely assign a value to the wrong attribute if we're having to look up / remember positions.

Reply:
Fair point.